app/index.html

```
@@ -803,6 +828,40 @@ <h3 class="section-sub-title">Insurance</h3>
                             </div>
                         </div>
 
+                        <!-- Spending CSV Upload -->
+                        <div class="card glass-card span-2">
+                            <h2 class="card-title">Spending Upload</h2>
+                            <p class="card-subtitle">Upload a Chase, Capital One, or generic (Date/Description/Amount) spending CSV — each transaction is auto-categorized and lands in the table below, fully editable.</p>
+                            <div class="import-drag-zone" id="spending-drag-zone">
+                                <p>Drag &amp; drop a spending CSV, or click to browse</p>
+                                <input type="file" id="spending-file-input" accept=".csv" style="display:none;">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)